ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -57,6 +59,39 @@ objects:
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
 
+    initContainers:
+    - name: share-installer-src
+      image: registry.svc.ci.openshift.org/${NAMESPACE}/pipeline:src
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      command:
+      - cp
+      - -R
+      - .
+      - /tmp/installer-src
+
+    - name: bazel-build-smoke
+      image: quay.io/coreos/tectonic-builder:bazel-v0.3
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: artifacts
+        # bazel needs to write to .cache
+        mountPath: /.cache
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+        export USER=bazel
+        cd /tmp/installer-src
+        mkdir /tmp/output
+        /usr/bin/bazel --output_base=/tmp/output build smoke_tests
+        cp bazel-bin/tests/smoke/linux_amd64_stripped/go_default_test /tmp/smoke
+        rm -rf bazel-bin
```

Comment:
I don't think we need to go out of our way to remove this.  I'm mostly interested in reducing the number of hoops we need to jump through to use Bazel.  Leaving a few files lying around does not bother me (these should be reaped automatically when the job completes, right).  But explicitly cleaning up after ourself isn't going to hurt either, so I'm fine if you want to leave this line in.